backend/compact-connect/README.md

```diff
@@ -98,13 +98,16 @@ To execute the tests, simply run `bin/run_tests.sh` from the `backend` directory
 ### First deploy to a Sandbox environment
```

Comment:
Since it's only twice, I'm fine repeating the HostedZone setup (in fact, I'd probably agree it's easier to read), but if it becomes a more frequent need, let's make it DRY.